bundles/org.openhab.binding.shelly/doc/images/AdvancedUsers.md

```diff
@@ -0,0 +1,94 @@
+# Advanced Users
+
+This section provides information for advanced use cases.
+
+## Additiona Resources
+
+There are additional resources available providing more information on Shelly devices and how to integrate those into openHAB:
+
+- [Shelly Homepage](https://shelly.cloud)
+- [Shelly Support Group (English)](https://www.facebook.com/groups/ShellyIoTCommunitySupport)
+- [Firmware Archive](http://archive.shelly-faq.de)
+- [API Documentation](https://shelly-api-docs.shelly.cloud/?fbclid=IwAR23ukCi_3aBSTPRHYUIcpr0pLi0vcyL0fF0PnJQdFvkkc8_Zo5LkAcli_A#http-server)
+
+and the [openHAB Community thread specifically for the binding](https://community.openhab.org/t/shelly-binding) - any feedback welcome here.
+You could also [report a bug or request a feature](https://github.com/openhab/openhab-addons/issues?q=is%3Aissue+is%3Aopen+%5Bshelly%5D) for the Shelly Binding.
+
+## Firmware Upgrade
+
+The Shelly App usually displays the installed firmware and also provide the function to upgrade the device with new firmware.
+However, if this doesn't work (sometimes there are issues) you you use the [Shelly Firmware Archi Link Generator](http://archive.shelly-faq.de), which provides download links to current, but also archived firmware files for all devices. 
+
+|Version|Notes                                                                                             |
+|-------|--------------------------------------------------------------------------------------------------|
+|1.5.7  |Minimum supported version. Older versions work in general, but have impacts to functionality (e.g. no events for battery powered devices). The binding displays a WARNING in the log if the firmware is older.|
+|1.6.x  |First stable CoIoT implementation. AutoCoIoT is enabled when firmware version >= 1.6 is detected. |
+|1.7.x  |Add additional status update values, fixes various issues                                         |
+|1.8.0  |Brings CoIoT version 2, which fixes a lot issues and gaps of version 1.                           |
+|1.9.2  |Various improvements, roller favorites, CoAP fixes                                                |
+
+
+There are 3 options available to perform the upgrade
+- The Shelly App usually detects when a new version becomes available and offers to do the upgrade within the UI (Web or App)
+- Alterco provides the [Shelly Firmware Archive Link Generator](http://archive.shelly-faq.de).
+This can be used to generate the upgrade link, which could be easily used to perform the upgrade on the cli-level having an Internet connection on that terminal (Shelly device doesn't require an Internet acces).
```

Comment:
```suggestion
This can be used to generate the upgrade link, which could be easily used to perform the upgrade on the cli-level having an Internet connection on that terminal (Shelly device doesn't require an Internet access).
```

Reply:
ok